lib/class-wp-rest-menu-items-controller.php

```diff
@@ -354,11 +354,7 @@ public function prepare_item_for_response( $post, $request ) {
 		}
 
 		if ( in_array( 'parent', $fields, true ) ) {
-			$data['parent'] = absint( $menu_item->post_parent ); // Same as post_parent, expose as integer.
-		}
-
-		if ( in_array( 'menu_item_parent', $fields, true ) ) {
-			$data['menu_item_parent'] = absint( $menu_item->menu_item_parent ); // Same as post_parent, expose as integer.
+			$data['parent'] = absint( $menu_item->menu_item_parent ); // Same as post_parent, expose as integer.
```

Comment:
Is this comment correct? I thought this was the parent menu item.

Reply:
No it is not.

Reply:
What is it then? Because essentially what we want to do is point at the parent menu item.